### ISSUE TITLE
Does not work with python3.10 and python3.11 

### ISSUE THREAD

**Issue by nikow** [up42/image-similarity-measures#46]
It looks like project is needs simple rebuild to work with Python 3.10 and Python 3.11

Python3.11:
```
(venv) [nikow@thinira tmp]$ python --version
Python 3.11.1
(venv) [nikow@thinira tmp]$ python -m pip install --prefer-binary image-similarity-measures
ERROR: Ignored the following versions that require a different python version: 0.0.1 Requires-Python >=3.6, <3.8; 0.1.1 Requires-Python >=3.6, <3.8; 0.1.2 Requires-Python >=3.6, <3.8; 0.2.2 Requires-Python >=3.6, <3.8; 0.3.0 Requires-Python >=3.6, <3.8; 0.3.1 Requires-Python >=3.6, <3.8; 0.3.3 Requires-Python >=3.6, <3.8; 0.3.4 Requires-Python >=3.6, <3.9; 0.3.5 Requires-Python >=3.6, <3.10
ERROR: Could not find a version that satisfies the requirement image-similarity-measures (from versions: none)
ERROR: No matching distribution found for image-similarity-measures
(venv) [nikow@thinira tmp]$
```

Python3.10:
```
(venv) [nikow@thinira tmp]$ python --version
Python 3.10.9
(venv) [nikow@thinira tmp]$ python -m pip install image-similarity-measures
ERROR: Ignored the following versions that require a different python version: 0.0.1 Requires-Python >=3.6, <3.8; 0.1.1 Requires-Python >=3.6, <3.8; 0.1.2 Requires-Python >=3.6, <3.8; 0.2.2 Requires-Python >=3.6, <3.8; 0.3.0 Requires-Python >=3.6, <3.8; 0.3.1 Requires-Python >=3.6, <3.8; 0.3.3 Requires-Python >=3.6, <3.8; 0.3.4 Requires-Python >=3.6, <3.9; 0.3.5 Requires-Python >=3.6, <3.10
ERROR: Could not find a version that satisfies the requirement image-similarity-measures (from versions: none)
ERROR: No matching distribution found for image-similarity-measures
(venv) [nikow@thinira tmp]$ 
```

**Comment by aruiz** [comment]
Came here to report the same issue

**Comment by Freed-Wu** [comment]
Any progress about this issue? Before this bug is fixed, I have to use <https://imagemagick.org/script/compare.php>.

**Comment by nekhtiari** [comment]
@dc-up42 sorry to bug you here, but I was trying to do this, but got 403 permission error (The user 'nikoogeo' isn't allowed to upload to project).  Could someone from UP42 do this rebuild?

**Comment by DylanBartels** [comment]
@Freed-Wu @aruiz @nikow 
This has been fixed in [v0.3.6](https://github.com/up42/image-similarity-measures/releases/tag/v0.3.6)
